homeassistant/components/skybell/binary_sensor.py

```diff
@@ -16,10 +19,22 @@
 
 SCAN_INTERVAL = timedelta(seconds=10)
 
-# Sensor types: Name, device_class, event
+
+class SensorEntryClass(NamedTuple):
+    """Metadata for sensor types."""
+
+    name: str
+    device_class: str
+    event: str
+
+
 SENSOR_TYPES = {
-    "button": ["Button", DEVICE_CLASS_OCCUPANCY, "device:sensor:button"],
-    "motion": ["Motion", DEVICE_CLASS_MOTION, "device:sensor:motion"],
+    "button": SensorEntryClass(
+        "Button", device_class=DEVICE_CLASS_OCCUPANCY, event="device:sensor:button"
```

Comment:
If you add a trailing comma, black will format it a bit better: Each keyword on a new line.

Reply:
small little detail, done.